src/schemas/28-8832/schema.js

```diff
@@ -1,84 +1,105 @@
-import _ from 'lodash';
+import { merge } from 'lodash';
 import definitions from '../../common/definitions';
 import schemaHelpers from '../../common/schema-helpers';
+import { countries, states50AndDC } from '../../common/constants';
+
+// patterns
+const textOnlyPattern = '^(?!\\s)(?!.*?\\s{2,})[^<>%$#@!^&*0-9]+$';
+const numberAndDashPattern = '^[0-9]*[-]*[0-9]*[-]*[0-9]*$';
+
+const currencyAmountPattern = '^\\d+(\\.\\d{1,2})?$';
 
 const schema = {
   $schema: 'http://json-schema.org/draft-04/schema#',
   title: 'EDUCATIONAL/VOCATIONAL COUNSELING APPLICATION (28-8832)',
   type: 'object',
   additionalProperties: false,
-  definitions: _.pick(definitions, ['dateRange']),
+  definitions: {},
   properties: {
-    applicantEmail: {
-      type: 'string',
-      format: 'email',
-    },
-    applicantRelationshipToVeteran: {
-      type: 'string',
-    },
-    seekingRestorativeTraining: {
-      type: 'boolean',
-    },
-    seekingVocationalTraining: {
-      type: 'boolean',
-    },
-    receivedPamphlet: {
-      type: 'boolean',
-    },
-    veteranServiceBranch: {
-      type: 'string',
-    },
-    divorceOrAnnulmentPending: {
-      type: 'boolean',
+    claimantInformation: {
+      type: 'object',
+      properties: {
+        fullName: definitions.fullName,
+        ssn: definitions.ssn,
+        vaFileNumber: definitions.vaFileNumber,
```

Comment:
Just to be on the safe side, given all of the recent changes with inflections, it might be worthwhile to change this to `VAFileNumber`.

Reply:
good idea, updated